howtos/generic_ci_artifactory.rst

```diff
@@ -58,3 +61,91 @@ However, you can gather and upload that information using other CI infrastructur
 .. code-block:: bash
 
     curl -X PUT -u<username>:<password> -H "Content-type: application/json" -T /tmp/build_info.json "http://host:8081/artifactory/api/build"
+
+Generating build info from lockfiles information 
+################################################
+
+1. To begin associating the build information to the uploaded packages the first thing
+is calling to the ``start`` subcommand of ``conan_build_info``. This will set the
+`artifact_property_build.name` and `artifact_property_build.name` properties in the
+:ref:`artifacts.properties<artifacts.properties>`.
+
+.. code-block:: bash
+
+    $ conan_build_info start MyBuildName 42
+
+2. Call Conan using :ref:`lockfiles<versioning_lockfiles>` to create information for the 
+`Build Info json format <https://github.com/jfrog/build-info>`_.
+
+.. code-block:: bash
+
+    $ cd mypackage
+    $ conan create . mypackage/1.0@user/stable # We create one package
+    $ cd .. && cd consumer
+    $ conan install . # Consumes mypackage, generates a lockfile
+    $ conan create . consumer/1.0@user/stable --lockfile conan.lock
+    $ conan upload * -c -r local # Upload all packages to local remotes
+
+3. Create build information based on the contents of the generated `conan.lock` lockfile and the
+information retrieved from the remote (the authentication is for the remote where you uploaded the
+packages).
+
+.. code-block:: bash
+
+    $ conan_build_info create buildinfo.json --lockfile conan.lock --user admin --password password
+
+
+The ``create`` subcommand has the following arguments:
```

Comment:
That kind of detail and command help should go to the reference somewhere. not here.